contracts/BalanceTrackerFixedPriceBase.sol

```diff
@@ -190,18 +189,41 @@ abstract contract BalanceTrackerFixedPriceBase {
         _locked = 1;
     }
 
-    function _withdraw(uint256 balance) internal virtual;
+    function _withdraw(address mech, uint256 balance) internal virtual;
+
+    /// @dev Processes mech payment by mech service multisig.
+    /// @param mech Mech address.
+    /// @return Mech payment.
+    /// @return Marketplace fee.
+    function processPaymentByMultisig(address mech) external returns (uint256, uint256) {
```

Comment:
This is required IMO - mech itself or multisig operating the mech are able to withdraw. Also facilitates testing a lot.